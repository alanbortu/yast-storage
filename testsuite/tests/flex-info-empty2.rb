# encoding: utf-8

# testedfiles: helper.rb

module Yast

  class TestClient < Client

    def main

      def setup1()
        setup_system("empty")

        setup_part_info(<<-EOT)
PARTITION  mount=/boot  size=512MB
PARTITION  mount=/      size=8GB
PARTITION  mount=swap   size=1GB
PARTITION  mount=/data  sizepct=100
EOT
      end

      def setup2()
      end

      def setup3()
      end

      Yast.include self, "helper.rb"

    end

  end

end

Yast::TestClient.new.main
