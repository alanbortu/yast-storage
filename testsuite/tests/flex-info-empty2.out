Dump	Proposal:
Dump	Create extended partition /dev/sda1 (298.09 GiB)
Dump	Create boot volume /dev/sda5 (517.72 MiB) with ext4
Dump	Create swap volume /dev/sda6 (1.00 GiB)
Dump	Create volume /dev/sda7 (288.57 GiB) for /data with btrfs
Dump	Create root volume /dev/sda8 (8.01 GiB) with btrfs
Dump	Create subvolume boot/grub2/i386-pc on device /dev/sda8
Dump	Create subvolume home on device /dev/sda8
Dump	Create subvolume opt on device /dev/sda8
Dump	Create subvolume srv on device /dev/sda8
Dump	Create subvolume tmp on device /dev/sda8
Dump	Create subvolume usr/local on device /dev/sda8
Dump	Create subvolume var/crash on device /dev/sda8
Dump	Create subvolume var/lib/mailman on device /dev/sda8
Dump	Create subvolume var/lib/named on device /dev/sda8
Dump	Create subvolume var/lib/pgsql on device /dev/sda8
Dump	Create subvolume var/log on device /dev/sda8
Dump	Create subvolume var/opt on device /dev/sda8
Dump	Create subvolume var/spool on device /dev/sda8
Dump	Create subvolume var/tmp on device /dev/sda8
Dump	
Dump	Target Map Excerpt:
Dump	device:/dev/sda label:msdos
Dump	device:/dev/sda1 fsid:0xf
Dump	device:/dev/sda5 fsid:0x83
Dump	device:/dev/sda6 fsid:0x82
Dump	device:/dev/sda7 fsid:0x83
Dump	device:/dev/sda8 fsid:0x83
Dump	
Dump	Proposal Feedback:
Dump	Cound not do snapshots.
